packages/react-components/src/icons/alumni-badge.tsx

```diff
@@ -1,18 +1,18 @@
 const alumniBadge = (
   <svg
-    width="24"
-    height="24"
-    viewBox="0 0 24 24"
+    width="16"
```

Comment:
All the icons should be 24x24, I don't think you should have changed it based on Figma. 
By changing this, all the places where alumni-badge is used will also be affected.
<img width="769" alt="image" src="https://user-images.githubusercontent.com/9281732/193831550-59345a7d-5dcd-4be7-a10a-0710c3c693ea.png">

Reply:
I thought we would have to ignore the whitespace around it since it's adding 8px of margin on it. I tested other places and it looked ok. Will revert it though. Thanks
<img width="435" alt="image" src="https://user-images.githubusercontent.com/3672764/193882894-91e09592-b14f-4f2c-8a61-a9de9dc41d22.png">